### PR TITLE
4 formulatepseudocode

### LINKED ISSUE #4: formulate pseudocode
can be written out in readme

### PR DESCRIPTION
**Pull request by walterjamesperry**
Hey hey, Stephanie. This a pull request to merge the branch into master.

Keep on JavaScripting!

### CODE REVIEW

**Comment by walterjamesperry** [comment]
Closing this pull request

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <header>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+        <link rel="stylesheet" href="assets/style.css">
+    </header>
+    <body>
+        <h1>Password Generator!</h1>
+        <section>
+            <h3>Click the button to generate a unique password.</h3>
```
Comment (eevargas) [inline index.html:10]: This should be `h2`. 
If h2 is too big, you can overwrite on our style sheet.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <header>
```
Comment (eevargas) [inline index.html:3]: the tag here should be `<head>`, `<header>` is a content tag to use on the body.

**Review by eevargas — COMMENTED** [review]
LGTM
I added two comments on your html.

**Pull request merged**